src/bf_dpu_update.py

```diff
@@ -333,6 +333,54 @@ def get_push_uri(self):
         return (multi_part_uri, deprecated_uri)
 
 
+    def get_update_service_state(self):
+        url = self._get_url_base() + '/UpdateService'
```

Comment:
seems that duplicate with get_simple_update_protocols(), can we merge it?

Reply:
Do you mean merge the those functions into one function?
I do not think so, the only duplicated one is the URL.   If merged, it may make code complicated.

Reply:
yes, not must, since it send same cmd later to fetch protocol.